docs/consensus/pot.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Proof-of-Time
+sidebar_position: 5
+description: Proof-of-Time consensus component
+keywords:
+    - Consensus
+    - randomness
+    - challenge
+---
+In addition to Proof-of-Space component described in previous section, Dilithium is secured by a Proof-of-Time (PoT) component. The chosen PoT algorithm is sequential AES, tuned for 1 proof per second. PoT is a measure against long-range attacks, and addresses unpredictability and dynamic availability issues. 
+The permissionless Proof-of-Work (PoW) used by Bitcoin remains the most robust consensus method in decentralized systems. However, mining requires massive energy expenditure to brute-force hash solutions, which has led to the introduction of various alternative consensus mechanisms. When transitioning to energy-efficient proof systems, like Proof-of-Stake and Proof-of-Space, however, several issues arise. 
+
+## Long-range attack
+
+Unlike in Proof-of-Work, the process of block production in Proof-of-Stake and Proof-of-Space-based
+blockchains is not physically constrained. This makes such protocols vulnerable to a _long-range attack_, when an attacker can produce, very quickly a chain all the way from genesis to the current time, and this chain can potentially be heavier than the current "canonical" chain.
+
+<!-- ![LongRangeAttack](../../../src/Images/Long-Range_Attack.png) -->
+
+A long-range attack refers to an adversary controlling enough resources at some point of chain life to rewrite a significant portion of the chain history. In PoW protocols like Bitcoin, this requires controlling over 50% of the total network hashrate for a sustained period of time which is infeasible in practice. However, long-range attacks remain a serious threat in alternative consensus protocols that do not rely on Proof-of-Work.
```

Comment:
```suggestion
```
This just repeats the previous paragraph.
If you decide to keep this text, please make a new paragraph for the following text.

Reply:
Rephrased it a bit and added new line

Reply:
I still prefer not to mention that Bitcoin needs 50% here, because in general generating an alternative chain in Bitcoin requires a long time, and we write above that a long-range attack is when an alternative chain can be formed very quickly (which is probably just a restriction). We can also change the definition, but I'm happy with the focus on it (otherwise, we should also consider these attacks against PoT).

Ultimately your call.